jrnl/EncryptedJournal.py

```diff
@@ -40,8 +40,11 @@ def open(self, filename=None):
         """Opens the journal file defined in the config and parses it into a list of Entries.
         Entries have the form (date, title, body)."""
         filename = filename or self.config["journal"]
-
+        dirname = os.path.dirname(filename)
         if not os.path.exists(filename):
+            if not os.path.isdir(dirname):
+                os.mkdir(dirname)
+                print(f"[Directory {dirname} created]")
```

Comment:
We usually print messages like this to `stderr` ([example](https://github.com/jrnl-org/jrnl/blob/develop/jrnl/util.py#L53)). That keeps these messages from accidentally ending up in someone's journal if they are piping the output.